### DISCUSSION TITLE
Implementing Different Attributes for Different Audio Tracks

### DISCUSSION THREAD

**Discussion by MattJCR** [WyattBlue/auto-editor#435] [1. Help] [answered]
Is it possible, or will there be a feature in the future to implement different attributes for different audio tracks? For example:

auto-editor multi-track.mov --edit "(or audio:stream=0 audio:threshold=-19dB, audio:stream=1 audio:threshold=-31dB)"

**Comment by WyattBlue** [comment, marked as answer]
We already have this. The syntax for this is:
```
auto-editor multi-track.mov --edit "(or audio:-19dB,stream=0 audio:-31dB,stream=1)"
```
